### ISSUE TITLE
invalid dtc error

### ISSUE THREAD

**Issue by abbasebriastro** [danieljprice/phantom#587]
I changed the density in sphereinbox (included power-low profile) and from a certain point (I mean a certain amount of power−index) onwards encountered the following error while running the program:
---> DELETING temporary dump .tmp <--
t =  0.21350E-07 dt =  1.934E-08 (force), np = 592481
FATAL ERROR! force: invalid dtc : dtc =  0.000E+00
Can anyone help me?

**Comment by danieljprice** [comment]
probably the density is infinite somewhere

**Comment by abbasebriastro** [comment]
Yes, to consider the power-law density for a molecular cloud core (according to observation) I used the "sphere_in_box" setup subroutine (Phantom SPH), In "set_sphere" for core, I have given it a density profile as rho=rho0*(r/r0)^-p. The problem that arises in the code (the program runs well, but it makes the density infinite at the center) is that if the limit r goes to 0 (r → 0), the density will be infinite.
Do you know where the problem comes from? Maybe you can help me!
